src/routes/users/handlers.js

```diff
@@ -25,6 +25,19 @@ export async function getPossibleCollaborators(req, res) {
   }
 }
 
+export async function getUserStatistics(req, res) {
+  try {
+    const user = await userById(await currentUserId(req, res));
+    const regions = user.permissions.map((permission) => permission.regionId);
+    const authorization = new UserPolicy(user);
+    const canWrite = regions.every((region) => authorization.canWriteInRegion(region));
```

Comment:
I think they only need to write in one region to show the full response, so this should be ```regions.some``` not ```regions.every```, right?

Reply:
Are you saying find the one region they have write permission in ?

Reply:
if they can write in one region, they should see the full set of statistics because they might have created reports

Reply:
Lets go over this when you are back.